versions/1/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -8,3 +8,7 @@ spec:
   properties:
     skip:
       $ref: "./../skip.spec.yml#/definitions/skip"
+    wait_for_data_timeout:
+      description: Short explanation for why test has been skipped.
```

Comment:
Umm, this description is unrelated? :thinking:

Reply:
My bad, copied this structure from the "skip" definition.